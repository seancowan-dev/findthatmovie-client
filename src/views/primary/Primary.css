.results {
    min-height: 40%;
    position: relative;
    top: -5px;
    display: flex;
    flex-direction: column;
    background-color: #22252c;
    z-index: 1;
}

.welcome-modal {
    width: 100%;
    height: 100%;
    min-height: 100%;
    position: absolute;
    left: 0;
    z-index: 1000;
    background-color: black;
    transition: ease-out .3s;
}

.welcome-modal-text {
    min-height: 100%;
    background-color: #3F3250;
    color: #c7c7c7;
    padding: 10px;
}

.welcome-modal-text > h1 {
    text-align: center;
}

.welcome-modal-text > button {
    display: block;
    position: relative;
    margin-left: auto;
    margin-right: 0;
    margin-top: 20px;
    margin-bottom: 20px;
}

.onscreen {
    top: 0;
}

.offscreen {
    top: -100%;
}