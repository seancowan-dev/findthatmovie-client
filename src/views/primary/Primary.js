import React from 'react';
import Search from '../../Search/Search';
import Results from '../../Results/Results';
import Nav from '../../Nav/Nav';
import { observer } from 'mobx-react';

const Primary = observer((props) => {
    return <>
        <Nav />
        <div className="parallax"></div>
        {<div className="search-form-container"><Search /></div>}
        <div className="results"><Results /></div> 
    </>
});

export default Primary;