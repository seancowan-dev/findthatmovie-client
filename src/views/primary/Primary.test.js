import React from 'react';
import { shallow } from 'enzyme';
import Primary from './Primary';

describe('| Primary View Test Object |', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Primary debug />);
    expect(component).toMatchSnapshot();
  });
});