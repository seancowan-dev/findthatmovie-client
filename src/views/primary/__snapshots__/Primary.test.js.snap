// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`| Primary View Test Object | should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
